app/controllers/wiki_controller.rb

```diff
@@ -399,10 +399,10 @@ def methods
       @notes = Node.where(status: 1, type: %w(page))
         .where('node.nid IN (?)', nids)
         .where('(type = "note" OR type = "page" OR type = "map") AND node.status = 1 AND (node.title LIKE ? OR node_revisions.title LIKE ? OR node_revisions.body LIKE ? OR term_data.name = ?)',
-          '%' + params[:topic] + '%',
-          '%' + params[:topic] + '%',
-          '%' + params[:topic] + '%',
-          params[:topic])
+               '%' + params[:topic] + '%',
```

Comment:
Hey, I was not able to push to your branch. I was able to fix this by having them on one line and having the like start immediately below `'(type = "note"`..... like so
```
'(type = "n......
'%' + params[:topic] + '%', '%' + params[:topic] + '%', '%' + params[:topic] + '%', params[:topic])
``

Reply:
Ok. Do I need to do something now?